course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -340,7 +340,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pot_temperature_constraint = iris.Constraint('air_potential_temperature',\n",
```

Comment:
as above :  this *does* need to be ..._potential...

Reply:
Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)